pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -196,5 +159,17 @@ func IsBootstrapComplete(configMapClient corev1listers.ConfigMapLister, staticPo
 			return false, nil
 		}
 	}
+
+	// check if etcd-bootsrap member is still present within the etcd cluster membership
+	memberHealth, err := etcdClient.MemberHealth(context.Background())
```

Comment:
why memberHealth and not plain listing? you're not even using the health result

Reply:
+1, fix on the way